extensions/wikia/GlobalNavigation/GlobalNavigationHelper.class.php

```diff
@@ -101,9 +101,10 @@ public function getMenuNodes() {
 
 		$exploreDropdownLinks = [];
 
-		$WAMLinkLabel = wfMessage( 'global-navigation-wam-link-label' );
+		$fandomUniversityLabel = wfMessage( 'global-navigation-fandom-university-link-label' );
 		$CommunityLinkLabel = wfMessage( 'global-navigation-community-link-label');
-		$exploreWikiaLabel = wfMessage( 'global-navigation-explore-wikia-link-label');
+		$exploreWikisLabel = wfMessage( 'global-navigation-explore-wikis-link-label');
+		$wikisLabel = wfMessage( 'global-navigation-wikis-label');
 
 		if ( $wgLang->getCode() === self::DEFAULT_LANG ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)